packages/web3-common/test/unit/formatters.test.ts

```diff
@@ -71,36 +69,17 @@ describe('formatters', () => {
 		});
 	});
 
-	describe('isPredefinedBlockNumber', () => {
-		it('should return true for "latest"', () => {
-			expect(isPredefinedBlockNumber('latest')).toBeTruthy();
-		});
-
-		it('should return true for "pending"', () => {
-			expect(isPredefinedBlockNumber('pending')).toBeTruthy();
-		});
-
-		it('should return true for "earliest"', () => {
-			expect(isPredefinedBlockNumber('earliest')).toBeTruthy();
-		});
-
-		it('should return false for "unknown"', () => {
-			expect(isPredefinedBlockNumber('unknown')).toBeFalsy();
-		});
-	});
-
```

Comment:
Moved to [web3-utils](https://github.com/ChainSafe/web3.js/blob/c7933b378db2e220ca0444f3481e1d0c4cb16d04/packages/web3-utils/test/unit/validation.test.ts#L203)